multiwallet.py

```diff
@@ -333,6 +335,19 @@ def _get_bip39_seed(is_testnet):
     return hd_priv
 
 
+def _get_key_record():
+    key_record_str = input(
+        blue_fg(
+            "Enter an xpub key record to blind in the format [deadbeef/path]xpub (any path will do): "
+        )
+    ).strip()
+    try:
+        return parse_partial_key_record(key_record_str=key_record_str)
+    except ValueError as e:
+        print_red(f"Could not parse entry: {e}")
+        return _get_key_record()
```

Comment:
Not a big fan of recursive loops like this. Generally harder to reason about and change later. A simple `while True` loop with a break works just as well.

Reply:
Good point.

I added this as [a separate github issue](https://github.com/buidl-bitcoin/buidl-python/issues/53), since the CLI apps use this convention throughout. To make a change like that, I want to add some sort of testing to the CLI apps so that we can make changes and know it's not breaking them (right now the apps just defer to the test coverage already in buidl, but that's not ideal).